packages/flutter/lib/src/widgets/debug.dart

```diff
@@ -114,8 +114,17 @@ bool debugPrintGlobalKeyedWidgetLifecycle = false;
 ///  * [debugProfileLayoutsEnabled], which does something similar for layout,
 ///    and [debugPrintLayouts], its console equivalent.
 ///  * [debugProfilePaintsEnabled], which does something similar for painting.
+///  * [debugProfileBuildsEnabledUserWidgets], adds events for user generated
+///    [Widget] build.
 bool debugProfileBuildsEnabled = false;
 
+/// Adds [Timeline] events for every user created [Widget] built.
```

Comment:
this should mention the default value (especially because it's true)

Reply:
The default value shows up in the docs at the bottom of the page where the declaration is shown.